website/docs/docs/dbt-ai/integrate-mcp-claude.md

```diff
@@ -13,20 +13,70 @@ Claude is an AI assistant from Anthropic with two primary interfaces:
 
 ## Claude Code
 
-You can set up Claude Code with both the local and remote `dbt-mcp` server. We recommend using the local `dbt-mcp` for more developer-focused workloads. 
+You can set up Claude Code with both the local and remote `dbt-mcp` server. We recommend using the local `dbt-mcp` for more developer-focused workloads.
 
 ### Setup with local dbt MCP server
 
 Prerequisites:
-- Have an `.env` file with your environment variables 
-- Local dbt-mcp setup
+- Completed the [local MCP setup](/docs/dbt-ai/setup-local-mcp)
+- Know your configuration method (OAuth, dbt Core or Fusion, or environment variables)
 
-1. Run the following command to add the MCP server to Claude Code:
+#### Using OAuth or environment variables directly
+
+The preferred method is to configure environment variables directly in Claude Code's configuration file without needing a separate `.env` file:
+
+```bash
+# First, add the MCP server
+claude mcp add dbt -- uvx dbt-mcp
+
+# Then edit the configuration to add environment variables
+claude mcp edit dbt
+```
+
+In the configuration editor, add your environment variables based on your use case:
+
+<Tabs>
+<TabItem value="CLI only">
+
+For <Constant name="core" /> or <Constant name="fusion" /> only (no dbt platform):
```

Comment:
i haven't tested it but where are they adding these env vars? in the settings.json file?